web/src/components/CardForm.js

```diff
@@ -0,0 +1,86 @@
+import { Alert, Button, Spinner } from '@cmsgov/design-system-core';
```

Comment:
The login, edit own account, admin edit account, and admin create account forms shared a ***bunch*** of code.  So I pulled out the common bits and put them here.

![](https://media.giphy.com/media/134iGJcqa8A9lC/giphy.gif)